roles/common/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+   - name: load settings
+     shell: |
+       source "{{ SCRIPTDIR }}/_settings.sh"
+       export LC_ALL="en_US.UTF-8"
+       export LANGUAGE="en_US.UTF-8"
+
+   - name: update system
+     apt: update_cache=yes upgrade=dist autoremove=yes autoclean=yes
+
+   - name: install helper tools
+     apt: package={{ item }} state=latest
+     with_items:
+       - python
```

Comment:
Python was installed in pre-tasks already, right?

Reply:
thanks, removed it